main.go

```diff
@@ -63,13 +62,7 @@ func handleJobs(jobsC <-chan context.Job) {
 func executeTasks(tasks []*context.Task, watcher *context.Watcher) {
 	for _, task := range tasks {
 		command := task.Command
-		tokens := strings.Split(command, " ")
-		name := tokens[0]
-		var cmdArg []string
-		if len(tokens) > 1 {
-			cmdArg = tokens[1:]
-		}
-		cmd := exec.Command(name, cmdArg...)
+		cmd := exec.Command("/bin/bash", "-c", command)
```

Comment:
So what you want is support of shell commands, not just Environment variables (you didn't include this detail in the #14 PR's description, it is however in the title of #7 which I missed).

The right (cross-platform) way for adding support of env vars is [`os.ExpandEnv`](http://golang.org/pkg/os/#ExpandEnv) (yeah, you don't even need to use regexps).

For subcommands I would go with the parsing of `(` and `)`. Not an easy fix, you'll have to use a stack data structure and parse & execute every subcommand one-by-one. But cross-platformability worth it.

For support of the shell commands, I would add a new key to the configuration file (what you've mentioned in your comment above). But definitely not break the work of the tool for users of other platforms.

Alternatively (if you do not have to use subcommands and shell commands too often), you can just create a shell script (e.g. `smth.sh`):
```bash
docker exec -it $(docker-compose ps -q $TEST) echo "hi"
```
And then use it as follows:
```
{
	"command": "sh ./smth.sh"
}
```